--- nss-3.13.6/mozilla/security/nss/lib/certdb/crl.c    2011-07-24 06:48:10.000000000 -0700
+++ our-nss/mozilla/security/nss/lib/certdb/crl.c       2012-10-29 16:42:54.272271060 -0700
@@ -2643,7 +2643,7 @@
     PRBool lockedwrite = PR_FALSE;
     SECStatus rv = SECSuccess;
     CRLDPCache* dpcache = NULL;
-    CERTRevocationStatus status = certRevocationStatusRevoked;
+    CERTRevocationStatus status = certRevocationStatusUnknown;
     CERTCRLEntryReasonCode reason = crlEntryReasonUnspecified;
     CERTCrlEntry* entry = NULL;
     dpcacheStatus ds;
@@ -2663,6 +2663,8 @@
         *revReason = reason;
     }

+   return SECSuccess;
+
     if (t && SECSuccess != CERT_CheckCertValidTimes(issuer, t, PR_FALSE))
     {
         /* we won't be able to check the CRL's signature if the issuer cert
--- nss-3.13.6/mozilla/security/nss/lib/certhigh/certvfy.c      2012-04-23 17:33:27.000000000 -0700
+++ our-nss/mozilla/security/nss/lib/certhigh/certvfy.c 2012-10-29 16:24:22.940336766 -0700
@@ -154,7 +154,7 @@
 SEC_CheckCRL(CERTCertDBHandle *handle,CERTCertificate *cert,
             CERTCertificate *caCert, int64 t, void * wincx)
 {
-    return CERT_CheckCRL(cert, caCert, NULL, t, wincx);
+       return SECSuccess;
 }

 /*
